Token.js

```diff
@@ -135,6 +138,9 @@ class Token {
 	}
 
 	size(newsize) {
+		// Clamp token size to min/max token size
+		newsize = clamp(newsize, this.MIN_TOKEN_SIZE, this.MAX_TOKEN_SIZE);
```

Comment:
I think we should clamp the upper bound to `$("#scene_map").width()` as long as that exists